### PR TITLE
Added full suite of services

### PR DESCRIPTION
**Pull request by rmcd-mscb**
added full list of services.  Some services with entry points crash when running ``` docker-compose up --build``` but ofp remains running and individual services can be called using ```docker exec <service>``` .  Still some things to work out.

### CODE REVIEW

**Comment by isuftin** [comment]
I'd suggest not including the .idea subdir and adding **/.idea to .gitignore unless there's a specific reason for adding this in?

File: app/nhm [hunk 1]
```diff
@@ -36,9 +36,9 @@ if [ ! -d /var/lib/nhm/NHM-PRMS_CONUS ]; then
     fi
 fi
 
-if [ -n "$NUMDAYS" ]; then
-    cd "$HOME"
-    /usr/local/bin/ofp
-else
-    sleep infinity
-fi
+# if [ -n "$NUMDAYS" ]; then
```
Comment (isuftin) [inline app/nhm]: Just delete this instead of commenting it out. If you need it again, it will be in version control

**Comment by isuftin** [comment]
Fr your output logs, you may consider either just logging to stdout since it will end up in docker logs anyway and/or logging to a specific subdir like /var/logs/nhm/ so that you can mount a host volume to /var/logs/nhm/ to capture the output to the host

File: app/Dockerfile [hunk 2]
```diff
@@ -52,7 +53,24 @@ RUN ln -s /var/lib/nhm /root/nhm
...
+# prms output to ncf script
+ADD out2ncf /usr/local/bin
+RUN chmod 744 /usr/local/bin/out2ncf
+
+# prms verifier script
+ADD verifier /usr/local/bin
+RUN chmod 744 /usr/local/bin/verifier
+
 # controller script
+RUN pwd
```
Comment (isuftin) [inline app/Dockerfile]: This is probably not necessary

File: app/nhm (hunk 1 above)
Comment (rmcd-mscb) [reply app/nhm]: Yes

File: app/Dockerfile (hunk 2 above)
Comment (rmcd-mscb) [reply app/Dockerfile]: Yes I agree - that was just a fudge to get the build to update the container.

**Comment by isuftin** [comment]
Alright. Feel free to make the changes and ping me on here when that's done. Should be pretty straightforward

**Comment by rmcd-mscb** [comment]
Thanks for your help Ivan

**Pull request merged**